codingTest_김준우/가운데 글자 가져오기.js

```diff
@@ -0,0 +1,12 @@
+function solution(s) {
+    let splitS =s.split("");
+    if (splitS.length%2===0)
+        {
+            answer = splitS[splitS.length/2-1] + splitS[splitS.length/2];
```

Comment:
```suggestion
            answer = splitS[(splitS.length/2)-1] + splitS[splitS.length/2];
```
가독성을 위해서 ( )를 써 주세요.

Reply:
> 가독성을 위해서 ( )를 써 주세요.

```suggestion
            answer = splitS[(splitS.length / 2) - 1] + splitS[splitS.length / 2];
```
띄어쓰기두여

Reply:
넵 알겠습니다